apiserver/plane/app/views/workspace/preference.py

```
@@ -0,0 +1,43 @@
+            return Response(WorkspaceHomePreference.objects.filter(user=request.user, workspace_id=workspace.id).values("user", "workspace", "sort_order", "is_enabled", "key"), status=status.HTTP_201_CREATED)  
+
+        home_preferences = [
+            WorkspaceHomePreference.HomeWidgetKeys.QUICK_LINKS, 
+            WorkspaceHomePreference.HomeWidgetKeys.RECENTS, 
+            WorkspaceHomePreference.HomeWidgetKeys.MY_STICKIES,
+            WorkspaceHomePreference.HomeWidgetKeys.NEW_AT_PLANE,
+            WorkspaceHomePreference.HomeWidgetKeys.QUICK_TUTORIAL
+        ]
+
+        for preference in home_preferences:
+            home_preference = WorkspaceHomePreference.objects.create(user_id=request.user.id, workspace_id=workspace.id, key=preference)
```

Ruff (0.8.2)

Line too long (136 > 88)

(E501)